presto-main/src/test/java/com/facebook/presto/operator/BenchmarkHashBuildAndJoinOperators.java

```diff
@@ -174,20 +164,52 @@ protected void initializeBuildPages()
         @Param({"0.1", "1", "2"})
         protected double matchRate;
 
+        @Param({"varchar", "bigint", "all"})
+        protected String outputColumns;
```

Comment:
I would drop "varchar" param. It doesn't tell us anything about performance changes more compared to "bigint" vs "all". Especially in the context of ubenchto, where we don't want to blob number of test cases.